docs/help-center/docs/warden-app/trade.md

```
@@ -0,0 +1,216 @@
+**AI Trading Terminal** is a place in Warden where you can open perpetual trading positions and analyze market data. To access the Terminal, do this:
+
+1. Log in: 👉 [Warden](https://app.wardenprotocol.org)
+2. Open the **Trade** tab.
+
+You'll see the Terminal interface, where you can do the following:
+
+1. Select token pairs
+2. [Manage the trading wallet](#manage-the-trading-wallet)
+3. [Analyze the market](#analyze-the-market)
+4. [Create an order](#2-create-an-order)
+5. [Manage your position](#3-manage-your-trade)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)